### PR TITLE
Support no baseline issue.

### PR DESCRIPTION
**Pull request by sima-zhu**
Js removes its baseline recently which failed the analysis step.
Failed pipeline: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1420643&view=logs&j=90227089-9f51-5ecc-566c-84e1f2918394&t=7da0f684-6369-5290-bb0b-5d27f59dde66
Need to add check for no baseline case.

Testing pipeline: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1421019&view=logs&j=bc67675d-56bf-581f-e0a2-208848ba68ca&t=7eee3a58-6120-518b-7fcb-7e943712aa81

### CODE REVIEW

**Review by jeremymeng — APPROVED** [review]
LGTM

File: eng/common/pipelines/templates/steps/credscan.yml [hunk 1]
```diff
@@ -35,7 +36,8 @@ steps:
 - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
   displayName: CredScan result analysis
   inputs:
-    GdnBreakBaselineFiles: ${{ parameters.BaselineFilePath }}
+    ${{if eq(variables['BaselineExists'], True)}}:
+      GdnBreakBaselineFiles: ${{ parameters.BaselineFilePath }}
```
Comment (weshaggard) [inline eng/common/pipelines/templates/steps/credscan.yml:40]: Let's see if we can just pass empty for this if the baseline file isn't needed.

Comment (sima-zhu) [reply eng/common/pipelines/templates/steps/credscan.yml:40]: https://github.com/Azure/azure-sdk-for-js/pull/20749/files

**Comment by sima-zhu** [comment]
We can simply remove the baseline parameter in js repo. Closed the PR as it is not needed.

**Pull request closed without merging**